packages/flutter_box_transform/lib/src/transformable_box.dart

```diff
@@ -597,7 +597,7 @@ class _TransformableBoxState extends State<TransformableBox> {
       content = GestureDetector(
         behavior: HitTestBehavior.translucent,
         supportedDevices: widget.supportedDragDevices,
-        onTap: onTap,
+        onTap: widget.onTap == null ? null : onTap,
```

Comment:
This change is unnecessary. `onTap` if null, passing it down as null explicitly is same as `onTap: onTap`.